workflow/scripts/compare_pathways.py

```diff
@@ -2,11 +2,11 @@
 import polars.selectors as cs
 import altair as alt
 
-
 diffexp_x = pl.read_csv(snakemake.input[0], separator="\t").lazy()
 diffexp_y = pl.read_csv(snakemake.input[1], separator="\t").lazy()
-label_x = list(snakemake.params.labels[0].keys())[0]
-label_y = list(snakemake.params.labels[1].keys())[0]
+label_x = list(snakemake.params.labels.keys())[0]
+label_y = list(snakemake.params.labels.keys())[1]
```

Comment:
As above, let's drop the `.keys()`:
```suggestion
label_x = list(snakemake.params.labels)[0]
label_y = list(snakemake.params.labels)[1]
```